src/main/java/no/unit/nva/testutils/HandlerUtils.java

```diff
@@ -0,0 +1,34 @@
+package no.unit.nva.testutils;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.node.ObjectNode;
+import java.io.ByteArrayInputStream;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import java.util.Map;
+
+public final class HandlerUtils {
+
+    private static final ObjectMapper OBJECT_MAPPER = new ObjectMapper();
+    public static final String BODY_FIELD = "body";
+    public static final String HEADERS_FIELD = "headers";
+
+    public static <T> InputStream requestObjectToApiGatewayRequestInputSteam(T requestObject,
+                                                                             Map<String, String> headers)
+        throws JsonProcessingException {
+        String requestString = requestObjectToApiGatewayRequestString(requestObject, headers);
+        return new ByteArrayInputStream(requestString.getBytes(StandardCharsets.UTF_8));
+    }
+
+    public static <T> String requestObjectToApiGatewayRequestString(T requestObject, Map<String, String> headers)
```

Comment:
We need support for setting request parameters

Reply:
fixed